Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -39,7 +39,7 @@
                     "StartupTimeoutSeconds" : 900
                 },
                 "Platform pc": {
-                    "tags": "tools,renderer,dx12,vulkan,null"
+                    "tags": "tools,renderer,dx12"
```

Comment:
Is this change intended?
Doesn't this cause processing assets targeting vulkan and the null renderer from the AP to fail?

Reply:
yes, this was mistakenly added